frontend/Cargo.toml

```
@@ -8,4 +8,9 @@ edition = "2021"
 [dependencies]
 yew = { version="0.20", features=["csr"] }
 yew-router = "0.17"
+reqwest = "0.11.13"
+web-sys = "0.3.60"
+prost = "0"
```

```suggestion
prost = { workspace = true }
```
Can we use the version from the workspace or it should be "0"? 🤔

---

I think the workspace version would do